ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1246,3 +1246,52 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  cluster: api.ci
+  cron: '@daily'
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  - base_ref: master
+    org: openshift
+    repo: ci-tools
```

Comment:
Why do we need this?

Reply:
we dont actually, we would already have the image - so not needed